cmd/tsbs_load_mongo/creator.go

```diff
@@ -68,6 +68,28 @@ func (d *dbCreator) CreateDB(dbName string) error {
 		return fmt.Errorf("create collection err: %v", res.Err().Error())
 	}
 
+	if timeseriesCollection && timeseriesCollectionSharded {
```

Comment:
It would be great if we rename the `timeseriesCollectionSharded` flag to `isCollectionSharded` or something like that, and change this condition to only use `if timeseriesCollectionSharded {`. This way, we can also have the option to test the scenario of sharding a manual bucketing or using a regular collection.